packages/lib/utils/validate.ts

```diff
@@ -0,0 +1,14 @@
+import z from "zod";
+import { ValidationError } from "@formbricks/errors";
+
+type ValidationPair = [any, z.ZodSchema<any>];
+
+export const validateInputs = (...pairs: ValidationPair[]): void => {
+  for (const [value, schema] of pairs) {
+    try {
+      schema.parse(value);
+    } catch (error: any) {
+      throw new ValidationError("Validation failed");
```

Comment:
Can we provide more information in the error?
I think it makes sense to also print the zod result of the validation to the console so we can debug if an error occurs. e.g. in `apps/web/app/api/v1/client/responses/[responseId]/route.ts` we do:
```ts
  const inputValidation = ZResponseInput.safeParse(responseInput);

  if (!inputValidation.success) {
    return responses.badRequestResponse(
      "Fields are missing or incorrectly formatted",
      transformErrorToDetails(inputValidation.error),
      true
    );
  }
  ````
Let's maybe also do a safeParse here and if `!inputValidation.success` console.error the error message and throw the validation error?